### PR TITLE
refactor(rxUrlUtils): rename urlUtils with rx prefix

### PR DESCRIPTION
**Pull request by lentusignavus**
Renames folder
Renames urlUtils service and tests

JIRA: https://jira.rax.io/browse/FRMW-1171

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/rxUrlUtils/scripts/rxUrlUtils.js [hunk 1]
```diff
@@ -128,4 +128,11 @@ angular.module('encore.ui.utilities')
         urlParser.href = url;
         return _.pick(urlParser, ['protocol', 'hostname', 'port', 'pathname', 'search', 'hash', 'host']);
     };
+})
+.service('urlUtils', function (rxUrlUtils) {
```
Comment (CITguy) [inline src/utilities/rxUrlUtils/scripts/rxUrlUtils.js]: Please add appropriate ngdoc documentation (with deprecation warning) for the `urlUtils` service.

File: src/utilities/rxUrlUtils/scripts/rxUrlUtils.spec.js [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+            var parsed = subject.parseUrl('http://something.com:1010');
+            expect(parsed.protocol).to.equal('http:');
+            expect(parsed.hostname).to.equal('something.com');
+            expect(parsed.port).to.equal('1010');
+            expect(parsed.host).to.equal('something.com:1010');
+        });
+    });
+});
+
+describe('urlUtils (DEPRECATED)', function () {
```
Comment (CITguy) [inline src/utilities/rxUrlUtils/scripts/rxUrlUtils.spec.js]: Test deprecated service to decrease likelihood of introducing unexpected bugs.

**Comment by cathysiller** [comment]
Dev LGTM

File: src/utilities/rxUrlUtils/scripts/rxUrlUtils.spec.js (hunk 2 above)
Comment (lentusignavus) [inline src/utilities/rxUrlUtils/scripts/rxUrlUtils.spec.js]: Weirdly enough I thought the idea came to me for `titleize` but not for this. Very subjective coding @lentusignavus. Shame on you

**Comment by lentusignavus** [comment]
I guess we need a 4th dev to LGTM this, since me and you touched the code @CITguy?

**Comment by parlarjb** [comment]
LGTM

**Comment by halkon** [comment]
LGTM

**Pull request merged**